tests/pipelines/text_to_video/test_text_to_video_zero.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+# Copyright 2023 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import torch
+
+from diffusers import TextToVideoZeroPipeline
+from diffusers.utils import require_torch_gpu, slow
+
+from ...test_pipelines_common import assert_mean_pixel_difference
+
+
+@slow
+@require_torch_gpu
+class TextToVideoZeroPipelineSlowTests(unittest.TestCase):
+    def test_full_model(self):
+        model_id = "runwayml/stable-diffusion-v1-5"
+        pipe = TextToVideoZeroPipeline.from_pretrained(model_id, torch_dtype=torch.float16).to("cuda")
+        generator = torch.Generator(device="cuda").manual_seed(0)
+
+        prompt = "A bear is playing a guitar on Times Square"
+        result = pipe(prompt=prompt, generator=generator).images
+
+        expected_result = torch.load("docs/source/en/api/pipelines/res/A bear is playing a guitar on Times Square.pt")
```

Comment:
```suggestion
        expected_result = torch.load("https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/text-to-video/A%20bear%20is%20playing%20a%20guitar%20on%20Times%20Square.pt")
```

Reply:
Let's not upload tensors and images directly to the GitHub repo

Reply:
Done!